policyengine_us/variables/gov/states/sc/tax/income/credits/cdcc/sc_cdcc.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class sc_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina CDCC"
+    documentation = "South Carolina Child and Dependent Care Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://dor.sc.gov/forms-site/Forms/IITPacket_2022.pdf#page=22"
+    )
+    defined_for = StateCode.SC
+
+    def formula(tax_unit, period, parameters):
+        # Get South Carolina CDCC rate
+        p = parameters(period).gov.states.sc.tax.income.credits.cdcc
+
+        # Get federal child care expenses
```

Comment:
This isn't a federally defined value.
```suggestion
        # Get child care expenses.
```